test/lib/cli.js

```diff
@@ -182,7 +184,7 @@ module.exports.getCloudConnectionStatus = async () => {
 
 module.exports.waitUntilOnline = async () => {
 	const { run } = module.exports;
-	const args = ['usb', 'cloud-status', DEVICE_ID, '--until', 'connected', '--timeout', 5 * 60 * 1000];
+	const args = ['usb', 'cloud-status', DEVICE_ID, '--until', 'connected', '--timeout', 5000];
```

Comment:
Waiting 5 seconds for the device to connect to the cloud during E2E tests isn't enough. Why the change?